jenkins_setup.sh

```diff
@@ -37,11 +37,11 @@ sudo cp -r jenkins_config/ /var/jenkins_home/
 echo "jenkins container id:"
 sudo docker run --name=jenkins \
      --env="VIRTUAL_PORT=8080" \
-     --env="VIRTUAL_HOST=192.168.33.5" \
+     --env="VIRTUAL_HOST=10.200.0.1" \
```

Comment:
I am tempted by 10.60.150.1, so we can reach it via alaska-gate?

Reply:
Sounds good. I will see if that works.